style.css

```diff
@@ -221,7 +248,7 @@ header {
 }
 
 .home-text {
-	background-color: rgba(255, 255, 255, 0.178);
+	background-color: var(--color-transparent-white);
 	padding: 30px;
 	border-radius: 10px;
```

Comment:
Can we add variables for border-radius? I think this will be used in a few places.

Reply:
Added and changed in the code